pkg/controller/risingwave_controller.go

```diff
@@ -336,6 +340,36 @@ func (c *RisingWaveController) reactiveWorkflow(risingwaveManger *object.RisingW
 	)
 	sharedSyncAllAndWait := ctrlkit.Shared(syncAllAndWait)
 
+	releaseScaleViewLock := mgr.NewAction(RisingWaveAction_ReleaseScaleViewLock, func(ctx context.Context, l logr.Logger) (ctrl.Result, error) {
+		risingWave := risingwaveManger.RisingWaveReader.RisingWave()
+		scaleViews := risingWave.Status.ScaleViews
+		aliveScaleView := make([]risingwavev1alpha1.RisingWaveScaleViewLock, 0)
+
+		for _, s := range scaleViews {
+			var scaleView risingwavev1alpha1.RisingWaveScaleView
+			err := c.Client.Get(ctx, types.NamespacedName{
+				Namespace: risingWave.Namespace,
+				Name:      s.Name,
+			}, &scaleView)
+
+			if err != nil {
+				if apierrors.IsNotFound(err) {
+					l.V(1).Info("Not found, Unlock")
```

Comment:
Shall we refine the log here? Looks like the lock isn't described 🤔 
```suggestion
					l.Info("Not found, unlock", "scaleview", s.Name)
```